spec/features/users/users_balances_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.feature 'Users::Balances', type: :feature do
+  let(:user) { create(:user) }
+  before do
+    10.times do
+      create(:transaction, user: user)
+    end
+    @balance = Users::Balance.new(user).balance
+  end
+
+  it 'calculates sum of transitions' do
+    expect(@balance).to eq(user.transactions.map { |transaction| transaction.amount.to_i }.sum)
```

Comment:
@marderer let's keep `it` block light -> move hard expectations to `let` blocks:
```ruby
let(:expected_balance) { user.transactions.sum(:amount).to_i }

it 'calculates sum of transitions' do
  expect(@balance).to eq(expected_balance)
end
```